entry_point.py

```diff
@@ -59,6 +61,66 @@ def change_values(file_name, getter_func):
         print(new_str.replace('\n', ''))
 
 
+def get_varname(var, prefix="ODOORC_"):
+    """
+    Cleans the string and return the variable ready to use in the config file
+
+    :param var: variable name as comes from the source
+    :param prefix: prefix to be deleted from the string
+    :return: Sting in lowercase without the prefix
+    """
+    r = re.compile("^"+prefix)
+    return r.sub("", var, 1).lower()
+
+
+def get_odoo_vars(getter_func):
+    res = dict()
```

Comment:
@ruiztulio It's faster to do:   

```python
res = {}
```

![image](https://user-images.githubusercontent.com/12100691/61065922-3f2ac680-a3ca-11e9-8294-5350017be1d9.png)